lib/src/web_socket/global_web_socket_monitor.dart

```diff
@@ -28,10 +26,10 @@ GlobalWebSocketMonitor newGlobalWebSocketMonitor() =>
     GlobalWebSocketMonitor._();
 
 WebSocketConnectEvent newWebSocketConnectEvent(
-        {@required String url,
-        @required bool wasSuccessful,
-        String sockJsSelectedProtocol,
-        List<String> sockJsProtocolsWhitelist}) =>
+        {required String url,
+        required bool? wasSuccessful,
```

Comment:
I think we can make this non-nullable if we update the calling code to use `late`